### PR TITLE
CF-597: add auth-uri if isAuthed=true

### PR DESCRIPTION
**Pull request by shintasmith**
Add auth-uri if isAuthed=true. Per Repose doc:
https://repose.atlassian.net/wiki/display/REPOSE/Client+Authentication+filter

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/rackerlabs/puppet-repose/pull/40" target="_blank">ReviewNinja

**Comment by shintasmith** [comment]
@mwhahaha, better?

File: templates/client-auth-n.cfg.xml.erb [hunk 1]
```diff
@@ -24,7 +24,11 @@
 <%- if @token_expire_feed -%>
     <atom-feeds check-interval="<%= @token_expire_feed['interval'] -%>">
         <rs-identity-feed id="identity-token-revocation-feed" uri="<%= @token_expire_feed['feed_url'] -%>" 
-                          isAuthed="true" user="<%= @token_expire_feed['user'] -%>" password="<%= @token_expire_feed['pass'] -%>"/>
+  <% if @token_expire_feed['use_auth'] %>
```
Comment (rackergs) [inline templates/client-auth-n.cfg.xml.erb]: need to use ```<%-``` and ```-%>```

File: templates/client-auth-n.cfg.xml.erb [hunk 2]
```diff
@@ -24,7 +24,11 @@
...
+                          isAuthed="true" auth-uri="<%= @token_expire_feed['auth_url'] -%>"
+                          user="<%= @token_expire_feed['user'] -%>" password="<%= @token_expire_feed['pass'] -%>"
+  <% end %>
```
Comment (rackergs) [inline templates/client-auth-n.cfg.xml.erb]: need to use ```<%-``` and ```-%>```

**Comment by shintasmith** [comment]
@rackergs, done

**Pull request merged**